### PR TITLE
Fix builds for newest artifacts/versions

### PR DESCRIPTION
**Pull request by ivy-lli**
- Raise engine version to v11.4.0
- Fix Variables / Form editor tests
- Add raise-build-plugin-version script
- Improve raise-version script, so it also raises the ivy engine version in maven.ts

### CODE REVIEW

File: playwright/tests/page-objects/variables-editor.ts [hunk 1]
```diff
@@ -12,8 +12,7 @@ export class VariablesEditor extends Editor {
   }
 
   override async isViewVisible() {
-    const header = this.viewFrameLoactor().locator('.master-toolbar > div > div');
-    expect(header).toHaveText('Variables Editor');
+    const header = this.viewFrameLoactor().locator('.master-toolbar:has-text("Variables Editor")');
```
Comment (ivy-lli) [inline playwright/tests/page-objects/variables-editor.ts:15]: fyi @ivy-lgi: the locator '.master-toolbar > div > div' lead to an error where the selector matched to two results (which cause an error in playwright). And an expect should always have an `await` as it is asynchron.

File: playwright/tests/form-editor.test.ts [hunk 2]
```diff
@@ -33,7 +33,7 @@ test.describe('Form Editor', () => {
   test('Edit input label', async () => {
     const input = editor.locatorFor('.block-input');
     await input.click();
-    const labelProperty = editor.locatorFor('[id=":re:-input"]');
+    const labelProperty = editor.locatorFor('#properties .ui-input').first();
```
Comment (ivy-lli) [inline playwright/tests/form-editor.test.ts:36]: fyi @ivy-lmu: ":re:" is a random id by react, so not really a good idea to use this in a test case :)

**Review by ivy-cst — APPROVED** [review]
Thanks a lot :+1:

**Review by ivy-rew — COMMENTED** [review]
great work; thanks @ivy-lli

**Comment by ivy-lmu** [comment]
thx 👍

**Pull request merged**